search-resources-api/src/test/resources/sample_opensearch_importCandidate_response.json

```diff
@@ -0,0 +1,97 @@
+{
+  "took": 49,
+  "timed_out": false,
+  "_shards": {
+    "total": 2,
+    "successful": 2,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 2,
+      "relation": "eq"
+    },
+    "max_score": 2.1928687,
+    "hits": [
+      {
+        "_index": "importCandidates",
+        "_type": "_doc",
+        "_id": "bd0f0ba3-e17d-473c-b6d5-d97447b26332",
+        "_score": 2.1928687,
+        "_source": {
+          "type": "ImportCandidate",
```

Comment:
Not sure if ImportCandidate has "type" property, because Publication has property "publicationType"